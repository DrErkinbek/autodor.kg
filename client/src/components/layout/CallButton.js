import React from 'react';

const CallButton = () => {
    return(
        <div className="fixed-action-btn">
            <a href="#call-modal" className="btn-floating btn-large blue darken-2 pulse modal-trigger">
                <i className="large material-icons">phone_in_talk</i>
            </a>
        </div>
    )
}

export default CallButton;