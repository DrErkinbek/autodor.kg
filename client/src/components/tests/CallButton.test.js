import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import CallButton from '../layout/CallButton';

it('CallButton renders without crashing ', () => {
    const div = document.createElement('div');
    ReactDOM.render(<CallButton />, div);
});

it('renders correctly and equal to snapshot', () => {
    const tree = renderer.create(<CallButton />).toJSON();
  
    expect(tree).toMatchSnapshot();
})