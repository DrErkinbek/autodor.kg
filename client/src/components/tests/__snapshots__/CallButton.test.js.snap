// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly and equal to snapshot 1`] = `
<div
  className="fixed-action-btn"
>
  <a
    className="btn-floating btn-large blue darken-2 pulse modal-trigger"
    href="#call-modal"
  >
    <i
      className="large material-icons"
    >
      phone_in_talk
    </i>
  </a>
</div>
`;
